### DISCUSSION TITLE
Moving all settings AND statistics to another board

### DISCUSSION THREAD

**Discussion by greymda** [hoylabs/OpenDTU-OnBattery#1103] [Q&A] [answered]
So I am planning to upgrade to a 16MB board and i have 2 questions:
1. can i move all my settings to the new board given its a 16MB S3 and my current one is a 4MB non-S3? like DPL settings, my inverter config, panel config and so on.

2. can i move all my stats? like inverter total yield, etc?

Cheers!

**Comment by spcqike** [comment, marked as answer]
1. Download your config file and re-upload it to your new module 

2. these statistics are reported by your inverter itself. So yes, they won’t get lost.
